src/systems/userinput/devices/keyboard.js

```diff
@@ -38,16 +38,18 @@ export class KeyboardDevice {
   }
 
   write(frame) {
-    this.events.forEach(event => {
+    for (let i = 0; i < this.events.length; i++) {
+      const event = this.events[i];
       if (event.type === "blur") {
         this.keys = {};
         this.seenKeys.clear();
-        return;
+      } else {
```

Comment:
Woops, good catch.

Reply:
This was fine before since the return in forEach just ends the "loop" early. Now that is a proper loop that returns from outer function